python_dwd/enumerations/parameter_enumeration.py

```diff
@@ -22,3 +22,22 @@ class Parameter(Enum):
     PRECIPITATION_MORE = "more_precip"
     PRESSURE = "pressure"
     CLIMATE_SUMMARY = "kl"
+
+
+PARAMETER_WORDLISTS = {
+    Parameter.TEMPERATURE_SOIL: [["soil", "boden", "ground"], ["temp"]],
+    Parameter.TEMPERATURE_AIR: [["air", "luft"], ["temp"]],
+    Parameter.PRECIPITATION: [["prec", "nied"]],
+    Parameter.TEMPERATURE_EXTREME: [["extr"], ["temp"]],
+    Parameter.WIND_EXTREME: [["extr"], ["wind"]],
+    Parameter.SOLAR: [["sol"]],
```

Comment:
Why you do not add `solar` here? Is this list only for shortcuts? If yes please rename the function

Reply:
The lists are for shortcuts, where I tried to find possible minimum words that could be entered by users. I'll rename the dictionaries accordingly.